Src/AutoFixture.xUnit.net2/CompositeDataAttribute.cs

```diff
@@ -0,0 +1,77 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Globalization;
+using System.Linq;
+using System.Reflection;
+using Xunit.Sdk;
+
+namespace Ploeh.AutoFixture.Xunit2
+{
+    /// <summary>
+    /// An implementation of DataAttribute that composes other DataAttribute instances.
+    /// </summary>
+    [AttributeUsage(AttributeTargets.Method, AllowMultiple = true, Inherited = true)]
+    [CLSCompliant(false)]
+    [System.Diagnostics.CodeAnalysis.SuppressMessage("Microsoft.Performance", "CA1813:AvoidUnsealedAttributes", Justification = "This attribute is the root of a potential attribute hierarchy.")]
+    public class CompositeDataAttribute : DataAttribute
+    {
+        private readonly IReadOnlyCollection<DataAttribute> attributes;
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositeDataAttribute"/> class.
+        /// </summary>
+        /// <param name="attributes">The attributes representing a data source for a data theory.
+        /// </param>
+        public CompositeDataAttribute(IReadOnlyCollection<DataAttribute> attributes)
+            : this(attributes.ToArray())
+        {
+        }
+
+        /// <summary>
+        /// Initializes a new instance of the <see cref="CompositeDataAttribute"/> class.
+        /// </summary>
+        /// <param name="attributes">The attributes representing a data source for a data theory.
+        /// </param>
+        public CompositeDataAttribute(params DataAttribute[] attributes)
+        {
+            if (attributes == null)
+            {
+                throw new ArgumentNullException("attributes");
+            }
+
+            this.attributes = attributes;
+        }
+
+        /// <summary>
+        /// Gets the attributes supplied through one of the constructors.
+        /// </summary>
+        public IEnumerable<DataAttribute> Attributes
+        {
+            get { return this.attributes; }
+        }
+
+        /// <summary>
+        /// Returns the composition of data to be used to test the theory. Favors the data returned
+        /// by DataAttributes in ascending order. Data already returned is ignored on next
+        /// DataAttribute returned data.
+        /// </summary>
+        /// <param name="methodUnderTest">The method that is being tested.</param>
+        /// <returns>
+        /// Returns the composition of the theory data.
+        /// </returns>
+        /// <remarks>
+        /// The number of test cases is set from the first DataAttribute theory length.
+        /// </remarks>
```

Comment:
This used to be the case but isn't any more - see https://github.com/AutoFixture/AutoFixture/blob/master/Src/AutoFixture.xUnit.net/CompositeDataAttribute.cs#L63